### ISSUE TITLE
A preview option?

### ISSUE THREAD

**Issue by dihuynh** [emichael/texthings#9]
I'm not sure how this can be done but I think it'll be a nice addition, since I'm not too pro at Latex.

**Comment by emichael** [comment]
What sort of preview are you thinking about? Like, a preview inside text fields?

**Comment by dihuynh** [comment]
Yes like inside the text fields. I'm not sure how feasible it is. I'm using the extension to type Latex in Facebook message. There's no way for you to see the expression before hitting "send". 

I was thinking you can start a little box when the user types $$ or [, then the box would enclose the expression as the user type. There can be a little eye that can toggle between Latex code and the compiled expression.

**Comment by emichael** [comment]
(Sorry for the late/practically non-existent reply.)

I don't think that's really a feasible thing to do inside arbitrary text fields on pages. Maybe a good alternative would be to do this inside `default_popup`? There could be a little textarea at the top and some JavaScript that updated a div below it and then refreshed MathJax every time the text field changed?
That way, the user always has a convenient place to fiddle with the LaTeX to get it right before copy-pasting it where they need it.

I don't have much time to work on this right now, but if you'd like to hack it together, I'd happily merge a pull-request and release it.
